packages/office-ui-fabric-react/src/components/Modal/Modal.styles.ts

```diff
@@ -22,7 +22,8 @@ export const getStyles = (props: IModalStyleProps): IModalStyles => {
     modalRectangleTop,
     theme,
     topOffsetFixed,
-    isModeless
+    isModeless,
+    layerProps
```

Comment:
Please pass in the exact thing you need, rather than objects. It will cache better.

Reply:
@dzearing I am using `layerProps && layerProps.className,` below, so all of `layerProps` is used. 

> The classname SHOULD be applied by the layer if the layerprops are being passed along.

If I have understood your [comment](https://github.com/OfficeDev/office-ui-fabric-react/pull/8273/files/50853adc8e908f88d18a71dfc8dec05f2afac772#r271009994) correctly, `layerProps` I believe is still needed. Please correct me if I am wrong.

Reply:
you just need layerClassName

Reply:
You do not NEED all of the layerProps. You just need the layerClassName. This function (`getClassNames`) should be taking immutable objects as input, so that we can cache results more efficiently. You are passing in a constantly mutating object here which would break caching. Please pull out exactly what you need.

Reply:
okay, shall submit a fix.